bc_obps/reporting/models/report_change.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+from registration.models.time_stamped_model import TimeStampedModel
+from reporting.models.report_version import ReportVersion
+from reporting.models.triggers import immutable_report_version_trigger
+from reporting.models.rls_configs.report_change import Rls as ReportChangeRls
+
+
+class ReportChange(TimeStampedModel):
```

Comment:
If we want to make this a separate model, we can probably call it `ReportReasonForChange` for readability on Metabase, with a matching table name.

But since this is a single field that has a one-to-one relationship to the `ReportVersion` model, I'd suggest making it an extra nullable/blankable field on the `ReportVersion` model itself instead.
This way, we can also simplify our API structure and add that field to the `/report-version/{version_id}` endpoint